src/services/currencyRates.js

```diff
@@ -0,0 +1,10 @@
+import { fetchWithTimeout } from '../utility';
+import { defaultCurrencyQuery } from '../utility/appConstants';
+
+const getCurrencyByQuery = (query = defaultCurrencyQuery) => {
+  return fetchWithTimeout(
+    `https://free.currconv.com/api/v7/convert?q=${query}&compact=ultra&apiKey=${process.env.REACT_APP_CURRENCY_API_KEY}`
+  ).then(resp => (resp ? resp.json() : {}));
```

Comment:
Try using modern async-await syntax for better readability.

Reply:
> Also, there is a rating component on Material UI, check that too.
> 
> https://material-ui.com/components/rating/

This is for when the user gives a rating and that has not been implemented yet.